app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
```

Comment:
You are importing multiple classes/modules in a single line. According to the checklist, it's better to use `( )` while importing multiple classes/modules. Here is how you can do it: `from app.errors import (NotWearingMaskError, VaccineError)` `from app.cafe import (Cafe)`